ci-operator/config/openshift/release/openshift-release-master__nightly-4.12.yaml

```diff
@@ -432,6 +432,12 @@ tests:
     - chain: ipi-aws-pre
     - ref: fips-check
     workflow: openshift-e2e-aws
+- as: e2e-gcp-ovn-xpn
+  interval: 48h
+  steps:
+    cluster_profile: gcp
+    workflow: openshift-e2e-gcp-xpn
+  timeout: 6h0m0s
```

Comment:
This is on 4.12, can we move it to the 4.13 nightly? 

I think the 4.12 feature gate may cause a problem, and we can just test it on 4.13.